object/slicer/slicer.go

```diff
@@ -581,6 +581,10 @@ func (x *PayloadWriter) _writeChild(ctx context.Context, meta dynamicObjectMetad
 		payload := obj.Payload()
 		payloadAsBuffers := [][]byte{obj.Payload()}
 
+		if uint64(len(payload)) > x.payloadSizeLimit {
```

Comment:
Link objects are flat simple ones only for now. Extension can be made in the future, but I doubt it's needed.

Reply:
ok, i was just wondering

Reply:
> @carpawell did u think about what will happen if we'll make recursion here?

that will require node-side adoption for sure, and it is hard to predict how much of a change it will take

> the 2nd question is what payload limit will be with x.payloadSizeLimit=64M?

1560671 parts can be attached to the payload, that is 95TB